.changeset/proud-days-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+"live-mobile": major
```

Comment:
@Justkant Not a major change. Let's pick a minor instead if you agree

Reply:
Yep, I asked for a patch at the time only to create the new pre-release, but I think it's not needed in the end.
I'll also cleanup the branch of all the commits but the first one before merging because we don't want all the pre-release stuff in develop